core/iwasm/aot/aot_runtime.c

```diff
@@ -1188,8 +1626,14 @@ aot_instantiate(AOTModule *module, AOTModuleInstance *parent,
     if (stack_size == 0)
         stack_size = DEFAULT_WASM_STACK_SIZE;
 #if WASM_ENABLE_SPEC_TEST != 0
-    if (stack_size < 48 * 1024)
-        stack_size = 48 * 1024;
+#if WASM_ENABLE_TAIL_CALL == 0
+    if (stack_size < 128 * 1024)
+        stack_size = 128 * 1024;
```

Comment:
@wenyongh 
why have you bumped stack size here?

Reply:
I can't remember it well now since it took a long time to develop the GC feature, seems that at first the tail call cases were disabled temporarily or not added to GC spec proposal, and running GC cases requires more native stack. But now the tail call is enabled when testing GC cases, maybe can restore it to 48KB now. I tested the value with some features and didn't find issue: `./test -s spec -t aot -b -P`, `./test -s spec -t aot -p -b -P`, `./test -s spec -t aot -M -b -P`, `./test -s spec -t aot -W -b -P`.